test/server/config-model.spec.js

```diff
@@ -92,7 +92,7 @@ describe('Config Model', () => {
                 return readFileSync(path, options);
             });
 
-            loadConfig();
+            config = loadConfig();
```

Comment:
aha!

Reply:
I almost commented inline that "this is the substantive change" 🤣